pkgs/k0sproject/k0s/registry.yaml

```diff
@@ -3,13 +3,41 @@ packages:
   - type: github_release
     repo_owner: k0sproject
     repo_name: k0s
-    asset: k0s-{{.Version}}-{{.Arch}}
-    format: raw
-    description: k0s - Zero Friction Kubernetes
-    supported_envs:
-      - linux
-    replacements:
-      x86_64: amd64
-      aarch64: arm64
-      armv7l: arm
-      armv8l: arm
+    description: k0s - The Zero Friction Kubernetes
+    version_filter: not (Version matches "-(alpha|beta|rc)")
+    version_constraint: "false"
+    version_overrides:
+      - version_constraint: semver("<= 0.8.1")
+        asset: k0s-{{.Version}}-{{.Arch}}
+        format: raw
+        supported_envs:
+          - linux
+      - version_constraint: semver("<= 1.27.16+k0s.0")
+        asset: k0s-{{.Version}}-{{.Arch}}
+        format: raw
+        windows_arm_emulation: true
+        supported_envs:
+          - linux
+          - windows
+      - version_constraint: "true"
+        asset: k0s-{{.Version}}-{{.Arch}}
+        format: raw
+        windows_arm_emulation: true
+        checksum:
+          type: github_release
+          asset: sha256sums.txt
+          algorithm: sha256
+        cosign:
+          signature:
+            type: github_release
+            asset: "{{.Asset}}.sig"
+          key:
+            type: github_release
+            asset: "cosign.pub"
+          opts:
+            # `--insecure-ignore-tlog` is required because it is signed with `--tlog-upload=false` option
+            # see also: https://github.com/k0sproject/k0s/blob/v1.32.3%2Bk0s.0/.github/workflows/release.yml#L106
+            - --insecure-ignore-tlog
```

Comment:
- https://github.com/k0sproject/k0s/commit/a01579db2dc7916194e6fd95949b6b4fbbfa2252
- https://github.com/k0sproject/k0s/pull/5724

Seems like this will be back ported to 1.29 or later.

Reply:
Supports all currently released versions.

https://github.com/aquaproj/aqua-registry/pull/35740/commits/54bee550cb44d030489f477fd75220316af9a1bc